MovieQuiz/Services/MovieQuizPresenter.swift

```diff
@@ -0,0 +1,137 @@
+import UIKit
+
+final class MovieQuizPresenter: QuestionFactoryDelegate {
+    let questionsAmount: Int = 10
+    private var questionNumberGlobal: Int = 0
+    var corrects: Int = 0
+    var currentQuestion: QuizeQuestion?
+    weak var viewController: (MovieQuizViewControllerProtocol)?// MovieQuizViewController?
```

Comment:
Скобки лишние, комментарий тоже можно убрать
```suggestion
    weak var viewController: MovieQuizViewControllerProtocol?// MovieQuizViewController?
```